src/containers/RTLPanel/index.test.js

```diff
@@ -1,6 +1,9 @@
 import React from 'react'
-import {shallow} from 'enzyme'
+import { shallow, configure } from 'enzyme'
 import RTLPanel from '.'
+import Adapter from 'enzyme-adapter-react-16'
+
+configure({ adapter: new Adapter() })
```

Comment:
Latest enzyme requires an adapter to work: https://airbnb.io/enzyme/docs/installation/index.html#working-with-react-16